internal/server/routes.go

```diff
@@ -103,6 +102,7 @@ func (s *Server) GenerateRoutes(promRegistry prometheus.Registerer) *gin.Engine
 	// TODO: remove after a couple versions
 	get(a, authn, "/v1/users/:id/grants", a.ListUserGrants)
 	get(a, authn, "/v1/groups/:id/grants", a.ListGroupGrants)
+	get(a, authn, "/v1/users/:id/groups", a.ListUserGroups)
```

Comment:
Oops, these should have been added as regular gin routes that don't add to the openAPI doc. I missed that. I'll fix that here.

Reply:
yeah I was saying above redirect rewrites could be made to support this case